internal/database/metadata/informer/informer.go

```diff
@@ -167,7 +168,7 @@ func (f *Informer) GetRevisionBy(ctx context.Context, groupID int, revision int6
 	if revision := f.Cache().Revisions.Find(func(r *types.Revision) bool {
 		return r.GroupID == groupID && r.Revision == revision
 	}); revision == nil {
-		return nil, fmt.Errorf("revision not found")
+		return nil, errdefs.NotFound(fmt.Errorf("revision not found"))
```

Comment:
revision %d not found

Reply:
maybe `revision not found by group %d and revision %d` better?